### PR TITLE
Retire essay fetch

### PR DESCRIPTION
**Pull request by myusuf**

### CODE REVIEW

File: core/management/commands/chronam_sync.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class Command(LoggingCommand):
     verbose = make_option('--verbose', action='store_true', dest='verbose', default=False, help='')
 
     skip_essays = make_option(
-        '--skip-essays', action='store_true', dest='skip_essays', default=False, help='Skip essay loading.'
+        '--skip-essays', action='store_true', dest='skip_essays', default=True, help='Skip essay loading.'
```
Comment (acdha) [inline core/management/commands/chronam_sync.py:19]: Wouldn't this prevent it from being enabled? I was thinking of flipping that to be `--no-skip-essays`.

Comment (myusuf) [reply core/management/commands/chronam_sync.py:19]: I thought having skip_essays to be True would always skip skip essay loading?

Comment (myusuf) [reply core/management/commands/chronam_sync.py:19]:
```if not options['skip_essays']:
            load_essays(settings.ESSAYS_FEED)```

Comment (myusuf) [reply core/management/commands/chronam_sync.py:19]: The double negative makes things a little confusing. Maybe we should consider using something like --load-essays to drive this behavior. I don't know

**Review by acdha — APPROVED** [review]

**Pull request merged**